public/config/external.js

```diff
@@ -1,2 +1,2 @@
 import 'bootstrap/js/dist/modal'
-// import 'bootstrap/dist/css/bootstrap.min.css'
```

Comment:
Are you sure? I don't know why we're using it but @johnyfarrar may do.

Reply:
this was something I'd originally put in, it's not required as the css is being imported via scss files